From be9f6a0dc90847dbb00307d23f47b8b3fc3ff130 Mon Sep 17 00:00:00 2001
From: Tudor Ambarus <tudor.ambarus@freescale.com>
Date: Fri, 29 May 2015 15:49:22 +0300
Subject: [PATCH 20/38] fix uninitialized variable compiler warning

crypto/../../cryptodev-linux/ioctl.c: In function 'cryptodev_compat_ioctl':
crypto/../../cryptodev-linux/ioctl.c:1445:2: warning: 'ret' may be used uninitialized in this function [-Wmaybe-uninitialized]
  return ret;
  ^

Signed-off-by: Tudor Ambarus <tudor.ambarus@freescale.com>
Change-Id: Id5226fc97a3bb880ca6db86df58957122bbaa428
Reviewed-on: http://git.am.freescale.net:8181/37441
Reviewed-by: Cristian Stoica <cristian.stoica@freescale.com>
Tested-by: Cristian Stoica <cristian.stoica@freescale.com>
---
 ioctl.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ioctl.c b/ioctl.c
index f3ce2f6..7cd3c56 100644
--- a/ioctl.c
+++ b/ioctl.c
@@ -1387,9 +1387,10 @@ cryptodev_compat_ioctl(struct file *file, unsigned int cmd, unsigned long arg_)
 	case COMPAT_CIOCASYMFETCHCOOKIE:
 	{
 		struct cryptodev_pkc *pkc;
-		int i = 0;
+		int i;
 		struct compat_pkc_cookie_list_s cookie_list;
 
+		ret = 0;
 		cookie_list.cookie_available = 0;
 
 		for (i = 0; i < MAX_COOKIES; i++) {
-- 
2.7.0

